src/simularium/VisData.ts

```diff
@@ -237,7 +237,8 @@ class VisData {
 
     private setupWebWorker() {
         this.webWorker = new Worker(
-            new URL("../visGeometry/workers/visDataWorker", import.meta.url)
+            new URL("../visGeometry/workers/visDataWorker", import.meta.url),
+            { type: "module" }
```

Comment:
Slight concern that this modification can affect other clients like simularium-website etc. Does it work across-the-board?

Reply:
I've tested the website and it seems fine. nbsv actually also had an issue on build that couldn't find the web workers. I'm not sure if this change would resolve it or not